js/matchingModel.js

```diff
@@ -222,7 +223,50 @@ define([
     */
     getResponseType: function() {
       return 'matching';
+    },
+
+    /**
+     * Creates a string explaining the answers the learner should have chosen
+     * Used by ButtonsView to retrieve question-specific correct answer text for the ARIA
+     * 'live region' that gets updated when the learner selects the 'show correct answer' button
+     * @return {string}
+     */
+    getCorrectAnswerAsText: function() {
+      const correctAnswerTemplate = Adapt.course.get('_globals')._components._matching.ariaCorrectAnswer;
+      const ariaAnswer = this.get('_items').map(item => {
+        var correctOption = _.findWhere(item._options, { _isCorrect: true });
```

Comment:
Why var?
```suggestion
        const correctOption = _.findWhere(item._options, { _isCorrect: true });
```

Reply:
force of habit after nearly 20 years of typing it?